CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 5.1.0
+
+DEPRECATIONS:
+
+* The OIDC config API endpoints are deprecated.
```

Comment:
Deprecations section is usually used for things that are newly labeled as deprecated but not yet removed.  For removals, they should go in "BREAKING CHANGES".   Since we aren't planning to rev the major version, I'd also mention that the changelog message.  I guess it's a slightly unorthodox situation.